application/controllers/sql_queries.py

```diff
@@ -24,3 +24,16 @@ def query_instrument_by_id(instrument_id, session):
 
 def query_instrument(session):
     return session.query(InstrumentModel)
+
+
+def query_instruments_form_type_with_different_survey_mode(survey_id, form_type, survey_mode, session):
```

Comment:
Could you put a comment here so someone at a glance can know what is it you're searching for?  Something like
' Searches for instruments for a survey that match the formtype and does NOT match the supplied survey_mode'